src/main/java/slow_sort/InsertionSort.java

```diff
@@ -0,0 +1,25 @@
+package slow_sort;
+
+import sort.Sorter;
+import java.util.List;
+
+public class InsertionSort implements Sorter {
+    @Override
+    public void sort(final List<Integer> nums) {
+        for (int i = 1; i < nums.size(); i++) {
+            int value = nums.get(i);
+            int index = i;
+            for (int j = i - 1; j >= 0; j--) {
+                if (value < nums.get(j)) {
+                    nums.set(index, nums.get(j));
+                    nums.set(j, value);
+                    value = nums.get(j);
+                }
+                else {
+                    break;
+                }
```

Comment:
I would revert if condition: `if (value >= nums.get(j)) { break }`
In that case you don't need else